update top toc (#76)
IBM/KubeflowDojo: OpenShift/README.md

# Kubeflow Deployment on OpenShift Container Platform

> - [Kubeflow Deployment on OpenShift Container Platform](#kubeflow-deployment-on-openshift-container-platform)

Reviewer: ```suggestion

```

  - [Deploy Kubeflow](#deploy-kubeflow)

>     - [1. Deploy with `kfctl` CLI](#1-deploy-with-kfctl-cli)

Reviewer: ```suggestion
    - [Deploy with `kfctl` CLI](#1-deploy-with-kfctl-cli)
```

>     - [2. Deploy with the Kubeflow Operator](#2-deploy-with-the-kubeflow-operator)

Reviewer: ```suggestion
    - [Deploy with the Kubeflow Operator](#2-deploy-with-the-kubeflow-operator)
```

  ***Approach #2:*** Since OpenShift Container Platform 4.x, the OpenShift web console integrates with the Operators catalog where users can search for operators and install with several simple clicks. Users can follow the guide in this [link](operator/README.md) to add the Kubeflow Operator to the Operators catalog on the OpenShift cluster and install the operator then deploy the Kubeflow from the OpenShift web console. 

## Deploy Kubeflow Pipelines with Tekton backend

>   For users who want to run [Kubeflow Pipelines with Tekton](https://github.com/kubeflow/kfp-tekton), KfDef configuration files are provided in [manifests](manifests) directory. Users should follow the [KFP-Tekton install instructions on OpenShift in this link](manifests/README.md) to deploy.

Reviewer: ```suggestion
  For users who want to run [Kubeflow Pipelines with Tekton](https://github.com/kubeflow/kfp-tekton), KfDef configuration files are provided in manifests directory. Users should follow the [KFP-Tekton on OpenShift install guide](manifests/README.md) to deploy.
```
